config/default_records/request_types/009_bioscan_request_types.yml

```diff
@@ -0,0 +1,24 @@
+#Request types for Bioscan
+---
+limber_bioscan_lysate_prep:
+  name: Bioscan Lysate Preparation
+  asset_type: Well
+  order: 1
+  request_class_name: CustomerRequest
+  for_multiplexing: false
+  billable: true
+  product_line_name: Bioscan
+  acceptable_purposes:
+    - LILYS-96 Stock
+limber_bioscan_library_prep:
+  name: Bioscan Library Preparation
+  asset_type: Well
+  order: 1
+  # TODO: is this correct request class?
```

Comment:
We are creating a PacBio library pool in this pipeline, so IlluminaHTP is not right.
Use generic LibraryCreation request maybe?

Reply:
Also update product line to match if we change it